docs/docs/frigate/installation.md

```diff
@@ -87,12 +87,16 @@ Running in Docker with compose is the recommended install method:
 
 The following officially supported builds are available:
 
-`ghcr.io/blakeblackshear/frigate:stable` - Standard Frigate build for amd64 & RPi Optimized Frigate build for arm64
+`ghcr.io/blakeblackshear/frigate:stable` - Standard Frigate build for amd64
 `ghcr.io/blakeblackshear/frigate:stable-standard-arm64` - Standard Frigate build for arm64
+`ghcr.io/blakeblackshear/frigate:stable-rpi` - Frigate build optimized for arm64 Raspberry Pi
```

Comment:
I'm not sure we want this since it will be confusing as the standard `stable` contains the rpi build for arm64

Reply:
The CI manually takes the amd64 standard build and the RPi build and creates a manifest for them so they are in the same image tag

Reply:
What you have for jetson makes sense although you'll likely need to update the ci to create those tags. 

Blake adds the stable- versions only